packages/server/lib/util/settings.js

```diff
@@ -111,25 +154,38 @@ module.exports = {
     })
   },
 
+  /**
+   * Ensures the project at this root has a config file
+   * that is readable and writable by the node process
+   * @param {string} projectRoot root of the project
+   * @param {object} options
+   * @returns
+   */
   exists (projectRoot, options = {}) {
     const file = this.pathToConfigFile(projectRoot, options)
 
+    debug('find out if "%s" exists', file)
+
     // first check if cypress.json exists
     return maybeVerifyConfigFile(file)
     .then(() => {
       // if it does also check that the projectRoot
       // directory is writable
       return fs.accessAsync(projectRoot, fs.W_OK)
     }).catch({ code: 'ENOENT' }, () => {
-      // cypress.json does not exist, we missing project
+      debug('no module error')
+      // cypress.config.js does not exist, completely new project
       log('cannot find file %s', file)
 
-      return this._err('CONFIG_FILE_NOT_FOUND', this.configFile(options), projectRoot)
-    }).catch({ code: 'EACCES' }, { code: 'EPERM' }, () => {
+      return this._err('CONFIG_FILE_NOT_FOUND', this.configFile(projectRoot, options), projectRoot)
+    }).catch({ code: 'EACCES' }, () => {
```

Comment:
Do we not need the `EPERM` check anymore? why not?

Reply:
... we NEED the EPERM check. good catch.

Reply:
Done in 13b603d